overlays/dp1/cnpg-butler-dp1-pooler-svc.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: usdf-butler-dp1-pooler-transaction-rw
+  namespace: dp1
+  annotations:
+    metallb.universe.tf/address-pool: sdf-rubin-ingest
+spec:
+  type: LoadBalancer
+  loadBalancerIP : 172.24.10.12
+  internalTrafficPolicy: Cluster
+  ports:
+  - name: pgbouncer
+    port: 5432
+    protocol: TCP
+    targetPort: 5432
+  selector:
+    cnpg.io/poolerName: usdf-butler-dp1-pooler-transaction-rw
+  sessionAffinity: None
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: usdf-butler-dp1-pooler-session-rw
```

Comment:
It's not clear we need the session pooler at all.  It's slightly more convenient for some kinds of debugging.  Maybe it's useful for replication?

Reply:
We don't need the session poolers.  In the future we may need a session pooler or a db service to use pgadmin or another admin tool, but that could be handled later.  

Also, the pooler-service should be removed.  The `serviceTemplate` creates this.  It was there because `serviceTemplate` is a newer feature in CNPG.